http/src/main/resources/reference.conf

```diff
@@ -268,6 +300,20 @@ azure {
      ]
      enabled = true
      database-enabled = false
+     chart-versions-to-exclude-from-updates = [
```

Comment:
same thing as above, maybe some comment, maybe some comment inline about where this comes from?